### PR TITLE
Us 161

### PR DESCRIPTION
**Pull request by rbeerma**
Per Rob, this has passed testing. This can be reviewed and merged if all looks good.

### CODE REVIEW

File: src/net/sf/memoranda/FilterToken.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package net.sf.memoranda;
```
Comment (amuseng1) [inline src/net/sf/memoranda/FilterToken.java]: Hey Rob, weed to add a file banner here per our coding standards.

Comment (amuseng1) [inline src/net/sf/memoranda/FilterToken.java]: Hey Rob, we need to add a file banner comment here per coding standards.

File: src/net/sf/memoranda/FilterToken.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+import net.sf.memoranda.date.CalendarDate;
+
+public class FilterToken {
```
Comment (amuseng1) [inline src/net/sf/memoranda/FilterToken.java]: We need a class banner comment here.

**Comment by rbeerma** [comment]
Good catch! I've added those banner comments and will initiate a new pull request.
Thanks!

**Pull request closed without merging**